src/lib/components/Popover/README.mdx

```diff
@@ -0,0 +1,294 @@
+        />
+        {isPopoverOpen && (
+          <Popover id="my-popover">
+            Hello there!
+          </Popover>
+        )}
+      </div>
+    </div>
+  );
+}}
+</Playground>
+
+See [API](#api) for all available options.
+
+## Placement
+
+By default, Popover is placed relative to the closest parent element with
+`position: relative` or `position: absolute`. Available placements are: top,
+right, bottom, and left. Additionally, all basic placements can be aligned to
+the center (default, no suffix), start (e.g. `top-start`), or end (e.g.
+`bottom-end`). Check Popover [API](#api) for the complete list of accepted
+values.
+
+<Playground>
+{() => {
+  const [align, setAlign] = React.useState('');
+  // All inline styles in this example are for demonstration purposes only.
+  return (
+    <>
+      <Toolbar align="baseline">
+        <ToolbarItem>
+          <span id="alignment-options-label">Alignment:</span>
+        </ToolbarItem>
+        <ToolbarItem>
+          <ButtonGroup aria-labelledby="alignment-options-label">
+            <Button
+              color={align === '-start' ? 'dark' : 'primary'}
+              label="start"
+              onClick={() => setAlign('-start')}
+            />
+            <Button
+              color={align === '' ? 'dark' : 'primary'}
+              label="center"
+              onClick={() => setAlign('')}
+            />
+            <Button
+              color={align === '-end' ? 'dark' : 'primary'}
+              label="end"
+              onClick={() => setAlign('-end')}
+            />
+          </ButtonGroup>
+        </ToolbarItem>
+      </Toolbar>
+      <div
+        style={{
+          display: 'grid',
+          placeContent: 'center',
+          minWidth: '20rem',
+          minHeight: '15rem',
+        }}
+      >
+        <div style={{ position: 'relative' }}>
+          <Placeholder bordered aria-describedby="my-popover-top">
+            Popovers
+            <br />
+            all day long…
+          </Placeholder>
+          <Popover id="my-popover-top" placement={`top${align}`}>
+            Top side
+          </Popover>
+          <Popover id="my-popover-right" placement={`right${align}`}>
+            Right side
+          </Popover>
+          <Popover id="my-popover-bottom" placement={`bottom${align}`}>
+            Bottom side
+          </Popover>
+          <Popover id="my-popover-left" placement={`left${align}`}>
+            Left side
+          </Popover>
+        </div>
+      </div>
+    </>
+  );
+}}
+</Playground>
+
+## Advanced Positioning
+
+While the basic setup can be sufficient in some scenarios, dropping a Popover
+usually won't be so easy. To handle all possible situations and edge cases
+automatically, including smart position updates, we recommend to involve an
+external library designed specifically for this purpose.
+
+ℹ️ The following example is using an external library [Floating UI] (formerly
```

Comment:
I would remove the `formerly
Popper.js` part. I dont think it brings any value and as time goes it will be less and less relevant.

Reply:
No strong opinion there, but people may know Popper.js mainly from Bootstrap which makes use of it. I can remove it if you insist.

Reply:
I do insist

_(I'm rarely givent the opportunity to insist, so I want to make the best of it)_ :smile_cat: